cou/apps/auxiliary.py

```diff
@@ -431,6 +431,51 @@ def upgrade_plan_sanity_checks(
 class OVNPrincipal(OVN):
     """OVN principal application class."""
 
+    def pre_upgrade_steps(
+        self, target: OpenStackRelease, units: Optional[list[Unit]]
+    ) -> list[PreUpgradeStep]:
+        """Pre Upgrade steps planning.
+
+        :param target: OpenStack release as target to upgrade.
+        :type target: OpenStackRelease
+        :param units: Units to generate upgrade plan
+        :type units: Optional[list[Unit]]
+        :return: List of pre upgrade steps.
+        :rtype: list[PreUpgradeStep]
+        """
+        steps = super().pre_upgrade_steps(target, units)
+        if self.config.get("enable-auto-restarts", {}).get("value") is False:
```

Comment:
**nitpick (non-blocking):** 

Can this be a property?

Reply:
same as below, I think in the future it won't be needed